plugins/inputs/opcua_listener/README.md

```diff
@@ -0,0 +1,176 @@
+  #
+  ## Authentication Method, one of "Certificate", "UserName", or "Anonymous".  To
+  ## authenticate using a specific ID, select 'Certificate' or 'UserName'
+  # auth_method = "Anonymous"
+  #
+  ## Username. Required for auth_method = "UserName"
+  # username = ""
+  #
+  ## Password. Required for auth_method = "UserName"
+  # password = ""
+  #
+  ## Option to select the metric timestamp to use. Valid options are:
+  ##     "gather" -- uses the time of receiving the data in telegraf
+  ##     "server" -- uses the timestamp provided by the server
+  ##     "source" -- uses the timestamp provided by the source
+  # timestamp = "gather"
+  #
+  ## Node ID configuration
+  ## name              - field name to use in the output
+  ## namespace         - OPC UA namespace of the node (integer value 0 thru 3)
+  ## identifier_type   - OPC UA ID type (s=string, i=numeric, g=guid, b=opaque)
+  ## identifier        - OPC UA ID (tag as shown in opcua browser)
+  ## tags              - extra tags to be added to the output metric (optional)
+  ## Example:
+  ## {name="ProductUri", namespace="0", identifier_type="i", identifier="2262", tags=[["tag1","value1"],["tag2","value2]]}
+  # nodes = [
+  #  {name="", namespace="", identifier_type="", identifier=""},
+  #  {name="", namespace="", identifier_type="", identifier=""},
+  #]
+  #
+  ## Node Group
+  ## Sets defaults for OPC UA namespace and ID type so they aren't required in
+  ## every node.  A group can also have a metric name that overrides the main
+  ## plugin metric name.
+  ##
+  ## Multiple node groups are allowed
+  #[[inputs.opcua.group]]
+  ## Group Metric name. Overrides the top level name.  If unset, the
+  ## top level name is used.
+  # name =
+  #
+  ## Group default namespace. If a node in the group doesn't set its
+  ## namespace, this is used.
+  # namespace =
+  #
+  ## Group default identifier type. If a node in the group doesn't set its
+  ## namespace, this is used.
+  # identifier_type =
+  #
+  ## Node ID Configuration.  Array of nodes with the same settings as above.
+  # nodes = [
+  #  {name="", namespace="", identifier_type="", identifier=""},
+  #  {name="", namespace="", identifier_type="", identifier=""},
+  #]
+
+  ## Enable workarounds required by some devices to work correctly
+  # [inputs.opcua.workarounds]
+    ## Set additional valid status codes, StatusOK (0x0) is always considered valid
+    # additional_valid_status_codes = ["0xC0"]
+```
+
+## Node Configuration
+
+An OPC UA node ID may resemble: "ns=3;s=Temperature". In this example:
+
+- ns=3 is indicating the `namespace` is 3
+- s=Temperature is indicting that the `identifier_type` is a string and `identifier` value is 'Temperature'
+- This example temperature node has a value of 79.0
+To gather data from this node enter the following line into the 'nodes' property above:
+
+```shell
+{field_name="temp", namespace="3", identifier_type="s", identifier="Temperature"},
+```
```

Comment:
This is not a shell script. Although it looks like a partial JSON, I guess it is meant to be a partial TOML?

Reply:
Something went wrong during the merge apparently.